book/caching.md

```diff
@@ -0,0 +1,33 @@
+Caching
```

Comment:
That's probably means guide should be divided into more smaller topics. But yeah, it makes sense to include it into cookbook as well.